src/Field/Gallery.php

```diff
@@ -29,7 +29,7 @@ class Gallery extends Image implements FieldInterface
     public function process($field)
     {
         $ids = $this->fetchValue($field);
-        $attachments = $this->post->whereIn('ID', $ids)->get();
+        $attachments = Post::whereIn('ID', $ids)->get();
```

Comment:
The db connection should be preserved.
```php
$connection = $this->post->getConnectionName();
$attachments = Post::on($connection)->whereIn('ID', $ids)->get();
```
Actually these connection problems are always happening. It should be interesting to write a test for that.